ertviz/views/selector_view.py

```diff
@@ -16,32 +16,44 @@ def parameter_selector_view(parent, data_type="parameter"):
                     ),
                     dbc.Col(
                         dcc.Input(
-                            id=parent.uuid("parameter-selector-filter"),
+                            id=parent.uuid(f"parameter-selector-filter-{suffix}"),
                             type="search",
                             placeholder="Substring...",
                             persistence="session",
                         ),
                         align="left",
                     ),
+                    dbc.Col(
+                        html.Button(
+                            id=parent.uuid(f"parameter-selector-button-{suffix}"),
+                            children=("Toggle selector visibility [On/Off]"),
+                        ),
+                    ),
                 ],
             ),
-            wcc.Select(
-                id=parent.uuid("parameter-selector-multi"),
-                multi=True,
-                size=10,
-                persistence="session",
-                className="ert-dropdown",
+            html.Div(
+                wcc.Select(
+                    id=parent.uuid(f"parameter-selector-multi-{suffix}"),
+                    multi=True,
+                    size=10,
+                    persistence="session",
+                    className="ert-dropdown",
+                ),
+                id=parent.uuid(f"container-parameter-selector-multi-{suffix}"),
+                # style={"display": "block"},
```

Comment:
could this be removed - or made more meaningful for others if it should stay as a future usage or something?